### DISCUSSION TITLE
YouTube RAG does not work anymore

### DISCUSSION THREAD

**Discussion by leuchtetgruen** [open-webui/open-webui#3186] [General]
# Bug Report

## Description

**Bug Summary:**
Asking questions about YouTube videos does not work as the embedded text is either "YouTube" or the contents of the YouTube website, but not the transcript.

**Steps to Reproduce:**

**Expected Behavior:**
It embeds the YouTube transcript and answers questions.

**Actual Behavior:**
It embeds the word YouTube or the website and answers "YouTube" to all questions

## Environment

- **Open WebUI Version:**  0.33
- **Ollama (if applicable):**  0.42

## Reproduction Details

**Confirmation:**

- [x ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by pkeffect** [comment]
Something must be wonky in your setup.

https://i.imgur.com/LGmlLnM.png

Seems to be working fine on my end.

**Comment by leuchtetgruen** [comment]
I get this message for all videos:

Something went wrong :/ Exception while accessing title of https://youtube.com/watch?v=9tcyWI45w7s. Please file a bug report at https://github.com/pytube/pytube

I have no idea what could be wrong
